docs/ionq/circuits.md

```diff
@@ -0,0 +1,118 @@
+# IonQ API Circuits
+
+To run against the IonQ API, you must construct circuits that are valid
+for the service. In other words, not every `cirq.Circuit` that you can
+construct will be able to run on the IonQ API, either against hardware
+or on the IonQ simulator.  Here we describe the restrictions on these circuits.
+
+In this section we assume a `cirq.ionq.Service` object has been instantiated and is
+called `service` and `cirq` and `cirq.ionq` have been imported:
+```python
+import cirq
+import cirq.ionq as ionq
+service = ionq.Service()
+```
+See [IonQ API Service](service.md) for how to set the service up.
```

Comment:
`set the service up.` - > `set up the service`

Reply:
done again :)